src/tbp/monty/frameworks/models/motor_policies.py

```diff
@@ -1896,7 +1887,7 @@ def avoid_revisiting_locations(
             # we might take
             # TODO could vectorize this
             rotated_locs = [
-                hab_utils.quat_rotate_vector(inverse_quaternion_rotation, point)
+                qt.rotate_vectors(inverse_quaternion_rotation, point)
```

Comment:
**note**: I think this could become `qt.rotate_vectors(inverse_quaternion_rotation, adjusted_prev_locs)` based on `qt.rotate_vectors` documentation: https://quaternion.readthedocs.io/en/latest/quaternion/#quaternion.rotate_vectors

Reply:
I ended up removing the now extraneous list comprehension as `qt.rotate_vectors` is intended to be applied to multiple vectors and will return the correct shape.